homeassistant/components/crownstone/strings.json

```
@@ -0,0 +1,30 @@
+{
+  "title": "Crownstone",
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::single_instance_allowed%]"
+    },
+    "error": {
+      "account_not_verified": "Account not verified. Please activate your account through the activation email from Crownstone.",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "email": "[%key:common::config_flow::data::email%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "description": "Enter your email and password for Crownstone.",
+        "title": "Crownstone"
```

Title is filled in automatically based on name in manifest. This description is not needed.
```suggestion
        }
```